src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewFilterMapper.xml

```diff
@@ -7,16 +7,43 @@
             <if test="studyViewFilter.studyIds != null and !studyViewFilter.studyIds.isEmpty()">
                 INTERSECT
                 SELECT sample_unique_id
-                FROM sample_view
+                FROM sample_mv
                 WHERE cancer_study_identifier IN
                 <foreach item="studyId" collection="studyViewFilter.studyIds" open="(" separator="," close=")">
                     #{studyId}
                 </foreach>
             </if>
+
+
+            <if test="studyViewFilter.genomicProfiles != null and !studyViewFilter.genomicProfiles.isEmpty()">
+                INTERSECT
+                SELECT * FROM (
+                <foreach item="ANDGroup" collection="studyViewFilter.genomicProfiles" separator="INTERSECT">
+                    SELECT sample_mv.sample_unique_id
+                    FROM sample_profile
+                    JOIN genetic_profile gp ON sample_profile.genetic_profile_id = gp.genetic_profile_id
+                    JOIN cancer_study cs ON gp.cancer_study_id = cs.cancer_study_id
+                    JOIN sample_mv on sample_profile.sample_id = sample_mv.internal_id
+                    <where>
+                        sample_mv.cancer_study_identifier IN
+                        <foreach item="studyId" collection="studyViewFilter.studyIds" open="(" separator="," close=")">
+                            #{studyId}
+                        </foreach>
```

Comment:
Do you need this filter? Since the filter for studyIds are above?

Reply:
@haynescd i m erged this PR but made follow up issues for this and the above comment